pkg/db/db_test.go

```diff
@@ -1339,3 +1341,82 @@ func TestDeleteRelease(t *testing.T) {
 		})
 	}
 }
+func TestReadWriteEslEvent(t *testing.T) {
+	const envName = "dev"
+	const appName = "my-app"
+	const lockId = "ui-v2-ke1up"
+	const message = "test"
+	const authorName = "testauthor"
+	const authorEmail = "testemail@example.com"
+	const evtType = EvtCreateApplicationVersion
+	expectedJson, _ := json.Marshal(map[string]interface{}{
+		"env":     envName,
+		"app":     appName,
+		"lockId":  lockId,
+		"message": message,
+		"metadata": map[string]string{
+			"authorName":  authorName,
+			"authorEmail": authorEmail,
+		},
+	})
+
+	tcs := []struct {
+		Name          string
+		EventType     EventType
+		EventData     *map[string]string
+		EventMetadata *map[string]string
+		ExpectedEsl   EslEventRow
+	}{
+		{
```

Comment:
there should be at least 2 test cases here.
One could be with Metadata and one with metadata=nil for example.